### PR TITLE
fix(cli): set start header

### PR DESCRIPTION
**Pull request by emhane**
Sets sync boundaries manually for headers and bodies downloaders in import command.

### CODE REVIEW

File: bin/reth/src/commands/import.rs [hunk 1]
```diff
@@ -145,6 +160,13 @@ impl ImportCommand {
             // override the tip
             let tip = file_client.tip().expect("file client has no tip");
             info!(target: "reth::cli", "Chain file chunk read");
+            // this will be start header of next chunk
+            let tip_header = file_client.tip_header().unwrap();
+
+            if start_header.is_none() {
+                // start header is genesis block
+                start_header = file_client.start_header();
+            }
```
Comment (rkrasiuk) [inline bin/reth/src/commands/import.rs:169]: genesis block should be imported out of pipeline, we should query file client for it

File: bin/reth/src/commands/import.rs [hunk 2]
```diff
@@ -136,6 +142,15 @@ impl ImportCommand {
...
         let mut reader = ChunkedFileReader::new(&self.path, self.chunk_len).await?;
 
+        let mut start_header: Option<SealedHeader> = None;
+
+        if self.start != 0 {
+            start_header = provider_factory
+                .provider()?
+                .sealed_header(block_num)
+                .expect("start block is not canonical with db");
+        }
```
Comment (rkrasiuk) [inline bin/reth/src/commands/import.rs:152]: we need to query for last available header in the storage (see how headers stage does it), no need for cli arg

**Review by rkrasiuk — CHANGES_REQUESTED** [review]

File: bin/reth/src/commands/import.rs (hunk 1 above)
Comment (emhane) [reply bin/reth/src/commands/import.rs:169]: hm, I'm not sure if that is possible, check this test out which I based myself on

https://github.com/paradigmxyz/reth/blob/6863cdb42bbac29de57b66739c8e0fc7b4d5dbaa/crates/net/downloaders/src/file_client.rs#L488

Comment (rkrasiuk) [reply bin/reth/src/commands/import.rs:169]: the test is correct because in the test the first header was set as local head

**Comment by emhane** [comment]
have tested this branch together with https://github.com/paradigmxyz/reth/pull/7777 and it works good

**Comment by rkrasiuk** [comment]
@emhane still unclear to me why `start` argument is needed at all

**Comment by emhane** [comment]
> @emhane still unclear to me why `start` argument is needed at all

will look into this

File: bin/reth/src/commands/import.rs [hunk 3]
```diff
@@ -206,6 +208,7 @@ impl ImportCommand {
         Ok(())
     }
 
+    #[allow(clippy::too_many_arguments)]
```
Comment (rkrasiuk) [inline bin/reth/src/commands/import.rs:211]: shouldn't be needed anymore
```suggestion
```

**Review by rkrasiuk — APPROVED** [review]
lgtm, PR description needs to be updated

**Pull request merged**